catalyst_voices/packages/internal/catalyst_voices_blocs/lib/src/campaign/info/campaign_info_cubit.dart

```diff
@@ -1,30 +1,72 @@
 import 'dart:async';
 
-import 'package:catalyst_voices_blocs/src/campaign/info/campaign_info.dart';
+import 'package:catalyst_voices_blocs/catalyst_voices_blocs.dart';
+import 'package:catalyst_voices_models/catalyst_voices_models.dart';
 import 'package:catalyst_voices_services/catalyst_voices_services.dart';
 import 'package:catalyst_voices_shared/catalyst_voices_shared.dart';
 import 'package:catalyst_voices_view_models/catalyst_voices_view_models.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 
 /// Gets the campaign info.
 final class CampaignInfoCubit extends Cubit<CampaignInfoState> {
-  final CampaignService campaignService;
+  final CampaignService _campaignService;
+  final AdminToolsCubit _adminToolsCubit;
```

Comment:
Does it make sense for `CampaignService` to emit mocked campaign when admin enables it so any other bloc widget is not aware or any of this trickery?

Reply:
I'll create an interface that AdminToolsCubit will implement so that we hide unnecessary functionalities from other cubits/blocs.

Unfortunately implementing admin tools in services layers is a bad idea and technically difficult because admin tools primarily are used for mocking UI. Services don't operate on UI level, they don't even have access to all view models which admin tools need to mock. If we wanted to do in service layers then we would need to bring UI logic / View models inside the services which I wanted to avoid.

Reply:
understand, i thought that `Campaign` may be mocked and it would be sufficient. I understand where you're coming from